Terasology/MetalRenegades
# feat: Generate hidden chests in the world

Adds randomly generated chests that spawn randomly on the surface. These chests are randomly filled with helpful items for the player. These chests can be further modified in code to give other items as well. 

<h3>Testing Instructions</h3>

- Create a new Metal Renegades world
- Explore until a chest block is found
- Open it, and it should contain around 1-6 of the following items:
  - Berries
  - Tools
  - Pistol/Bullets
